src/pages/news/news-detail/news-detail.js

```diff
@@ -49,10 +55,9 @@ const NewsDetailPage = ({ match }) => {
   const newsDate = dateToShow.toLocaleString(dateLanguage, TIME_OPTIONS);
   const newsImage = article.images ? article.images.primary.medium : ' ';
   const newsText =
-    article.text.length !== 0
+    article.text.length !== 0 && article.text[language].value !== null
```

Comment:
Do we need this check if we have a check at lines `#43`

Reply:
No, it was an old check. I forgot to delete it. Deleted